lib/internal/crypto/cipher.js

```diff
@@ -73,11 +75,11 @@ function getUIntOption(options, key) {
 function createCipherBase(cipher, credential, options, decipher, iv) {
   const authTagLength = getUIntOption(options, 'authTagLength');
 
-  this._handle = new CipherBase(decipher);
+  this[kHandle] = new CipherBase(decipher);
   if (iv === undefined) {
```

Comment:
Since the purpose of this is to hide the actual implementation details, I would prefer to use a non-enumerable symbol. Otherwise it will still show up in the repl and when inspecting the value (including console calls).

Reply:
I had the idea to do this after reviewing https://github.com/nodejs/node/pull/22684. If we want to use non-enumerable symbols, maybe we should add that to the guidelines?